apps/backoffice-v2/src/pages/Statistics/Statistics.page.tsx

```diff
@@ -1,5 +1,24 @@
-import React, { FunctionComponent } from 'react';
+import React, { FunctionComponent, useEffect, useState } from 'react';
+import { useZodSearchParams } from '@/common/hooks/useZodSearchParams/useZodSearchParams';
+import { DateRange, dateRangeSchema } from '@/common/hooks/useZodSearchParams/dateRangeSchema';
 
 export const Statistics: FunctionComponent = () => {
-  return <div>statistics page</div>;
+  const [dateRange, setDateRange] = useZodSearchParams(dateRangeSchema);
+
+  useEffect(() => {
+    dateRange && console.log(dateRange);
+  }, [dateRange]);
+
+  const fetchstatistics = async (from: DateRange['from'], to: DateRange['to']) => {
```

Comment:
Using an object  here would be less sensitive to the args positions and provide autocomplete and extra documentation of the function's signature.
```suggestion
  const fetchStatistics = async ({ from, to }: DateRange) => {
```